package.json

```diff
@@ -14,13 +14,15 @@
     "aphrodite": "^2.4.0",
     "axios": "^0.21.1",
     "babel-eslint": "^10.1.0",
+    "bootstrap": "^5.3.0",
     "classnames": "^2.2.6",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.5.0",
     "eslint-config-airbnb": "^18.1.0",
     "express": "^4.17.1",
     "formik": "^2.1.4",
+    "jquery": "^3.7.0",
```

Comment:
One question though What are we using jquery for?

Reply:
I added it because the nav bar wasnt active....but i think it was my dev environments issue....let me clean those sections out